server/athenian/api/align/queries/metrics.py

```diff
@@ -194,6 +203,62 @@ def _parse_time_interval(params: Mapping[str, Any]) -> Intervals:
     return goal_dates_to_datetimes(date_from, date_to)
 
 
+def _parse_requested_metrics(params: Mapping[str, Any]) -> Sequence[_RequestedMetric]:
+    metrics = params.get(MetricParamsFields.metrics)
+    metrics_w_params = params.get(MetricParamsFields.metricsWithParams)
+
+    if not (bool(metrics) ^ bool(metrics_w_params)):
+        fields = [MetricParamsFields.metrics, MetricParamsFields.metricsWithParams]
+        raise ResponseError(
+            InvalidRequestError(
+                pointer=" or ".join(f".{f}" for f in fields), detail=f"Use {' or '.join(fields)}",
+            ),
+        )
+
+    if metrics:
+        return [_RequestedMetric(m, None, None) for m in metrics]
+
+    assert metrics_w_params
```

Comment:
We can remove this assertion with 100% confidence, we've just checked `!=` and `metrics`.

Reply:
need it for mypy

Reply:
can you please add `# mypy` to these assertions because I am not using it and would remove this line automatically while refactoring smth.